src/ck-core/ckrdma.C

```diff
@@ -1295,12 +1284,6 @@ void CkRdmaEMBcastAckHandler(void *ack) {
 #if CMK_REG_REQUIRED
         deregisterMemFromMsg(myMsg, false);
 #endif
-        // send a message to the parent to signal completion
-        envelope *env = (envelope *)bcastInterimAckInfo->msg;
-        CkUnpackMessage(&env); // Unpack message before sending it to getParentBcastAckInfo
-        sendAckMsgToParent(env);
-        CkPackMessage(&env);
-
```

Comment:
Removal of sending ack message after the completion of all nodes below me in the tree.  (for bcast send api).